cli/bin/setup

```diff
@@ -1,2 +1,14 @@
 #!/usr/bin/env bash
+
 npm install --loglevel error --depth 0
+(cd ../exo-add && bin/setup && node_modules/.bin/build && echo 'exo-add √')
```

Comment:
Since this is setting up all subprojects, let's move this script to the root folder.